src/contrib/persistence/Akka.Persistence.Sqlite/Journal/BatchingSqliteJournal.cs

```diff
@@ -0,0 +1,119 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="BatchingSqliteJournal.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Configuration;
+using System.Data.Common;
+using System.Data.SQLite;
+using Akka.Configuration;
+using Akka.Persistence.Sql.Common.Journal;
+
+namespace Akka.Persistence.Sqlite.Journal
+{
+    public sealed class BatchingSqliteJournalSetup : BatchingSqlJournalSetup
```

Comment:
I should create constructor working with HOCON config. So far, it's up to implementing class to provide config-value mappings. By default all SQL implementations I've created, are working with default sql persistence config (so it's compatible with existing SQL settings). New config values have defaults, in case if user didn't specify them.